packages/nextjs/pages/index.tsx

```diff
@@ -28,6 +47,20 @@ const Home: NextPage = () => {
           </p>
         </div>
 
+        {/* Showing Account Status */}
+        {isInallowList !== undefined ? (
+          <div className="text-center">
+            <p className={isInallowList ? "text-green-400" : "text-red-400"}>
+              Account status: {isInallowList ? "Allowed" : "Not allowed"}
+            </p>
+            {isInallowList && (
+              <p>Check-in Status: {checkedInContract?.includes("0x0000") ? "Not Checked In" : "Checked In"}</p>
```

Comment:
This could fail if the contract starts with `"0x0000"`. Really hard for this to happen, but let's do the full zero address comparison === / !== to be safe.

You could also abstract that logic if you want to make it look cleaner,  something like:

```js
const isZeroAddress = (address) => {
  return address === "0x0000000000000000000000000000000000000000";
}
```

Ethers had that constant, but I think `viem` doesn't have it.

Reply:
I like this function idea, this will make code looks much cleaner and readable. Thanks